client/src/__test__/utils/RussianUtils.test.ts

```diff
@@ -0,0 +1,67 @@
+import { describe, expect, it } from 'vitest';
+import { translateTimeMessage, translateServerResponse } from '../../utils/TranslatorUtils';
+import { serverResponses } from '../../constants/serverResponses';
+
+describe('Portuguese Utils unit tests', () => {
+  it('should translate all time ago messages to ru', () => {
+    expect(translateTimeMessage('2 years ago', 'ru')).toBe('2 года назад');
+    expect(translateTimeMessage('1 year ago', 'ru')).toBe('1 год назад');
+    expect(translateTimeMessage('2 months ago', 'ru')).toBe('2 месяца назад');
+    expect(translateTimeMessage('1 month ago', 'ru')).toBe('1 месяц назад');
+    expect(translateTimeMessage('2 days ago', 'ru')).toBe('2 дня назад');
+    expect(translateTimeMessage('1 day ago', 'ru')).toBe('1 день назад');
+    expect(translateTimeMessage('2 hours ago', 'ru')).toBe('2 часа назад');
+    expect(translateTimeMessage('1 hour ago', 'ru')).toBe('1 час назад');
+    expect(translateTimeMessage('2 minutes ago', 'ru')).toBe('2 минуты назад');
+    expect(translateTimeMessage('1 minute ago', 'ru')).toBe('1 минуту назад');
+    expect(translateTimeMessage('2 seconds ago', 'ru')).toBe('2 секунд назад');
+    expect(translateTimeMessage('1 second ago', 'ru')).toBe('1 секунду назад');
+    expect(translateTimeMessage('Moments ago', 'ru')).toBe('Несколько минут назад');
+    expect(translateTimeMessage('lala', 'ru')).toBe('lala');
+    expect(translateTimeMessage('null', 'ru')).toBe('null');
+  });
+
+  it('should translate all time left messages to ru', () => {
+    expect(translateTimeMessage('2 years left', 'ru')).toBe('осталось 2 лет');
```

Comment:
```suggestion
    expect(translateTimeMessage('2 years left', 'ru')).toBe('осталось 2 года');
```

Reply:
Also, why some lines here are capitalized and some are not?

Reply:
Done here https://github.com/ricardo-campos-org/react-typescript-todolist/pull/189/commits/12b10e04de236beb52c8d9e180fc721ef5c1ff6e